services/ai-navigator-app/0.2.2/helmrelease/helmrelease.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-ai-navigator-cluster-info-api-charts
         namespace: kommander-flux
-      version: 0.1.0
+      version: 0.2.1
```

Comment:
why is the app version 0.2.2 but chart version is 0.2.1? Can we match the app version to be 0.2.1 also?

Reply:
on `main` the k-app is `0.2.2` while the chart version is `0.2.1`. for 2.7 I guess yes we can make k-app version 0.2.1 if we want

Reply:
yeah let's do that. especially if `main` (or 2.8) has config only changes on top of `0.2.1` then we will want 2.7 to have a lower version so the upgrade is not triggered automatically (if 2.7 and 2.8 have the same app version).

Reply:
done